.travis.yml

```diff
@@ -9,7 +9,9 @@ branches:
 
 cache:
   directories:
-    - $HOME/.composer/cache
+    - $HOME/.composer/
```

Comment:
Because we want to cache the whole tree. :smile:
On May 6, 2016 4:28 PM, "Frank Brückner" <notifications@github.com> wrote:

> In .travis.yml
> <https://github.com/zendframework/zend-session/pull/40#discussion_r62393399>
> :
>
> > @@ -9,7 +9,9 @@ branches:
> >
> >  cache:
> >    directories:
> > -    - $HOME/.composer/cache
> > +    - $HOME/.composer/
>
> Why is cache removed?
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly or view it on GitHub
> <https://github.com/zendframework/zend-session/pull/40/files/e0ac3caffc02b3cf6d1f14bd917efcfc6bf26ddb#r62393399>
>